### PR TITLE
Manual configuration in README.

### PR DESCRIPTION
**Pull request by YooSunYoung**
- Document deployment configuration
- Package deployment secret configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,31 @@ tox -e docs
...
 git remote add origin git@github.com:scipp/myproject.git
 ```
+
+## Manual Configuration
+Once the project is built from the template, there are manual settings to be configured per project/repository for deployment of documentation and packages.
+
+### Document Deployment
+1. Select branch of deployed document.
+  The document will be deployed from a branch via github action.
+  The branch of the document should be selected manually on github.
```
Comment (SimonHeybrock) [inline README.md:22]:
```suggestion
1. Select branch of deployed documentation.
  The documentation will be deployed from a branch via GitHub action.
  The branch of the documentation must be selected manually on GitHub.
```

File: README.md [hunk 2]
```diff
@@ -12,3 +12,31 @@ tox -e docs
...
+
+1. Conda
+Go to Settings > Secrets > Actions > Organization secrets.
+There is `scipp` organization-wide anaconda key, `ANACONDATOKEN`. But it should be enabled per repository.
+
+2. Pypi
+Go to Settings > Secrets > Actions > Repository secrets.
+`PYPI_TOKEN` should be configured per repository.
+You can make one under your account on (`pypi.org`)[https://pypi.org/].
+If it is the first time of deployment, the token needs to have all-access and after the first deployment, it can be replaced with the new token that has the access to the project.
```
Comment (SimonHeybrock) [inline README.md:42]: This was very confusing for me. I think I had to create a personal token? And the first upload was done manually with my username and password. Is there a better way? In any case, may be worth describing in more detail.

File: README.md [hunk 3]
```diff
@@ -12,3 +12,31 @@ tox -e docs
...
+1. Select branch of deployed document.
+  The document will be deployed from a branch via github action.
+  The branch of the document should be selected manually on github.
+  Go to `Settings > Pages` and set `source` as  `Deploy from a branch` and `Branch` as `gh-pages`.
+
+2. Add `.nojekyll` file.
+  Once the document is deployed to a branch, make sure it has `.nojekyll` file in the `gh-pages` branch and you should be able to see the documentation at `scipp.github.io/project_name`.
+
+3. Deployment key.
+  The deployment key should be already set by organization if it is under `scipp` organization.
```
[the remark this answers is not in the document]
Comment (YooSunYoung) [reply README.md:29]: Sorry I made it confusing, an `ssh key` was needed to deploy it to the `branch`. 
And, you're right, the deployment of documentation from branch is done by github.

**Comment by SimonHeybrock** [comment]
Another thing to add:

On the repo page, in the right sidebar, "About" section: Click the gear and check the "Use your GitHub Pages website" checkbox for "Website"

**Comment by YooSunYoung** [comment]
> Another thing to add:
> 
> On the repo page, in the right sidebar, "About" section: Click the gear and check the "Use your GitHub Pages website" checkbox for "Website"

The checkbox is not available in [beamlime](github.com/scipp/beamlime)... so I suppose it should be done after the first deployment of documentation...?

**Comment by SimonHeybrock** [comment]
> > Another thing to add:
> > On the repo page, in the right sidebar, "About" section: Click the gear and check the "Use your GitHub Pages website" checkbox for "Website"
> 
> The checkbox is not available in [beamlime](github.com/scipp/beamlime)... so I suppose it should be done after the first deployment of documentation...?

I suppose? Or after enabling GH-pages?

**Comment by YooSunYoung** [comment]
> > > Another thing to add:
> > > On the repo page, in the right sidebar, "About" section: Click the gear and check the "Use your GitHub Pages website" checkbox for "Website"
> > 
> > 
> > The checkbox is not available in [beamlime](github.com/scipp/beamlime)... so I suppose it should be done after the first deployment of documentation...?
> 
> I suppose? Or after enabling GH-pages?

Yes it was after selecting branch...! I updated the how-to

**Review by SimonHeybrock — APPROVED** [review]

**Pull request merged**